services/adapter.ts

```diff
@@ -8,21 +8,21 @@ import * as Bluebird from 'bluebird';
 
 // String.normalize is very large polyfill not included by default in core-js
 // and most likely we're not gonna need it
-if (!String.prototype.normalize) {
+if (typeof String.prototype.normalize !== 'function') {
   // eslint-disable-next-line no-extend-native
   String.prototype.normalize = function normalize() {
     return String(this);
   };
 }
 
 // Monkey-patch fetch Promise with Bluebird's.
-// @ts-ignore
+// @ts-expect-error
```

Comment:
`ts-expect-error` is used in unit tests (e.g. we want to test runtime type assertions).

i think `ts-ignore` is more relevant.

Reply:
I like the reasoning behind the [`prefer-ts-expect-error`](https://typescript-eslint.io/rules/prefer-ts-expect-error/) rule.

Reply:
i won't be persistent; it's not *that* important.